src/garnet/helpers/settings.py

```diff
@@ -0,0 +1,39 @@
+"""Global settings for Garnet."""
+from contextlib import contextmanager
+
+from mantid.kernel import ConfigService
```

Comment:
Are garnet settings intended to be put in `~/.mantid/Mantid.user.properties` or somewhere else? I understand that logging settings require interacting with mantid, but I would have expected there to be save/load somewhere else.